project/Doc.scala

```diff
@@ -88,6 +88,12 @@ object UnidocRoot extends AutoPlugin {
     case _ => Seq("--allow-script-in-comments")
   }
 
+
+  private def packageListLocation(baseDir:File, name:String) = {
+    baseDir / "docs"/"src"/"main"/"resources"/"package-list"/ name
```

Comment:
Manually maintained list. :-(

Reply:
What is the reason of having this manually maintained list? See https://github.com/lagom/lagom/pull/1283#discussion_r177371461

Reply:
`javadoc` supports adding cross-references with (at least) two methods:

* `-link` gets a remote base URI and `javadoc` will handle everything
* `-linksource` gets a remote base URI and a `package-list` 

The trick is that `-link` locates `package-list` for you so you don't have to manually maintain it. 

For some reason I couldn't narrow `-link` worked with `docs.oracle.com`:

`+      "-link", "https://docs.oracle.com/javase/8/docs/api/",`

but didn't work for our docs (`akka.io`, `playframework`) despite `package-list` being in the appropriate location on all cases.

I haven't dug deeper but it could be related to a missing header on the response of [https://doc.akka.io/japi/akka/current/package-list](https://doc.akka.io/japi/akka/current/package-list) causing `javadoc` to consider the downloaded file invalid.

Reply:
😍

Reply:
cc @marcospereira